src/Game.cpp

```diff
@@ -14,3 +19,53 @@ int Game::convertSingleCharToPoints(char input)
 
     throw std::logic_error("Invalid frame");
 }
+
+void Game::vectorOfDigits(const std::vector<int>& vec) {}
+
+int Game::summingOfPointsWithoutBonus(const std::vector<int>& vec)
+{
+    if(vec.size()!=0){
+        int sum = std::accumulate(vec.begin(), vec.end(), 0);
+              return sum;
+    }
+        throw std::logic_error("some data are required");
+ }
+
+int Game::summingOfBonusPoints(const std::vector<int>& vec){
+    std::vector<int> vec2;
+    std::vector<int> vec3;
+    int j=0;
+    for (int i=0; i<vec.size()-2; i++){
```

Comment:
formatowanie. Przed i po operatorach stawiaj spacje. To ogólna uwaga do całości.
`for (int i = 0; i < vec.size() - 2; i++) {`